### PR TITLE
[RDY] add konami code feature

### PR DESCRIPTION
**Pull request by hanfi**

### CODE REVIEW

**Comment by RidaBiki** [comment]
la vérité c'est beau !

**Comment by izeau** [comment]
👍 for the feature. 👎 for the implementation. Please provide a konami code snippet instead of requiring it from `node_modules`

File: index.html [hunk 1]
```diff
@@ -64,5 +64,12 @@
         </p>
       </div>
     </div>
+    <script src="https://cdn.jsdelivr.net/npm/konami@1.6.0/konami.min.js"></script>
```
Comment (deuzu) [inline index.html]: checksum plz

File: index.html [hunk 2]
```diff
@@ -64,5 +64,12 @@
...
+    <script>
+    var easter_egg = new Konami(function() {
```
Comment (deuzu) [inline index.html]:
```javascript
new Konami(() => {
    const audio = new Audio('assets/audio/screaming_goat.mp3');
    audio.play();
});
```

**Comment by deuzu** [comment]
Reverting :-1: to :+1: when checksum is ok
See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

File: index.html (hunk 1 above)
Comment (izeau) [reply index.html]: Also reset `package-lock.json` changes.

**Review by izeau — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (hanfi) [reply index.html]: putin j'aime beaucoup comment tu écris ton js !!! toi t'es pas un charlatan 

ah it should be in english : 
"bitch, i love too much how you write your js !!!you, you are not a charlatan"

**Comment by deuzu** [comment]
@hanfi Why is the package-lock.json has a diff if you're using a CDN?

**Pull request merged**